makefiles/application.inc.mk

```diff
@@ -3,4 +3,6 @@ MODULE = $(APPLICATION_MODULE)
 DIRS += $(RIOTCPU)/$(CPU) $(RIOTBOARD)/$(BOARD)
 DIRS += $(RIOTBASE)/core $(RIOTBASE)/drivers $(RIOTBASE)/sys
 
+BLOBS = $(APPLICATION_BLOBS)
```

Comment:
Why not move this line to `makefiles/blob.inc.mk` ? Since this file is included by Makefile.base, it will be more consistent to handle this in the related file. This is untested so maybe this doesn't work because I'm missing something.

Reply:
If that line would be in blob.inc.mk, it would be used in every recursive make call (e.g., for every module). We only want it to be used for the application module. All other modules can directly use BLOBS.

From the corresponding commit description:

```

@kaspar030
make: pass BLOBS to makefiles/application.inc.mk

For regular modules, adding files to BLOBS is sufficient to create the
corresponding headers.

Application modules are different, as they use a minimal makefile
(makefiles.application.inc.mk) to build, thus application level
variables are not available.

This commit makes Makefile.include pass BLOBS to the application
Makefile as APPLICATION_BLOBS, and application.inc.mk use that variable
as value for BLOBS.

The indirection is necessary so submakefiles (e.g., those visited by
DIRS) do not hard override BLOBS.
```

Reply:
Is it possible to add a comment about that here instead of inside the commit message ?

Reply:
Now it is here, is that sufficient?

Reply:
I've added some doc to applications.inc.mk.